src/components/UserInformationContainer/UserInformationContainer.tsx

```diff
@@ -31,6 +31,7 @@ export type UserInformationComponentProps = {
   isSignedIn: boolean
   isSigningIn: boolean
   address?: string
+  sliceAddressBy?: number
```

Comment:
This is not used in the storybook?

Reply:
the story written uses the default value